packages/pouch-storage/src/PouchCommandHistoryStorage.ts

```diff
@@ -25,7 +25,7 @@ export class PouchCommandHistoryStorage implements CommandHistoryStorage {
   }
 
   async getTable(language: string): Promise<PouchCommandHistoryTable> {
-    return new PouchCommandHistoryTable(language);
+    return this.getUpdateTable(language);
```

Comment:
Seems this will not do what I was hoping. I'll need to figure out a different way to disable db change events while bulk deleting.